astro/src/components/JSON.astro

```diff
@@ -21,7 +21,7 @@ const parsedTitle = title ? parseInline(title) : '';
 const code = JSON.stringify(json.data, null, 2);
 ---
 {/* title is optional, do not render an empty em tag */}
-{title && <em set:html={parsedTitle}/>}
+{title && <p class="mb-0"><em set:html={parsedTitle}/></p>}
```

Comment:
You didn't mean to but you tripped over something that was making me nuts back when we did the migration that I could not reconcile with a "tailwind-only" approach. This tightens up the title to the block for the JSON component but won't match if someone adds a title to a triple backtick block which was nearly impossible to target with prose.

Example:
<img width="1186" alt="image" src="https://github.com/FusionAuth/fusionauth-site/assets/11809481/2983e664-91e0-4c37-8b20-5f27e27e9b6a">

I'm adding the magic incantation to fix that to the global stylesheet so that these things match, but that's not my favorite place for it, really it should go in the "replace prose" stylesheet I need to make. I will open an issue for that.

We aren't using that much in the docs at the moment, but we should because it's neat